src/components/MainPage.js

```diff
@@ -152,22 +204,34 @@ class MainPage extends Component<Props, State> {
                         >
                           IMDB score: {item.vote_average}
                         </Typography>
+                        <Typography
+                          gutterBottom
+                          variant="headline"
+                          component="h4"
+                        >
+                          Release date: {item.release_date}
+                        </Typography>
                       </CardContent>
                       <CardActions>
                         <IconButton
                           className={classes.button}
                           aria-label="Delete"
-                          disabled
                           color="primary"
-                          onClick={this.deleteMovieFromFavorites}
+                          onClick={this.deleteMovieFromFavorites.bind(

```

Comment:
No. This is bad. You will recreate this function on every render. Also, you will update props of this component on every render.

Pls, use a class method instead here and in all other places in render.

Reply:
Fixed